### ISSUE TITLE
can't farm：Failed to open (invalid plots)

### ISSUE THREAD

**Issue by kihcl** [chiabluepool/bluepool#4]
I use windows version,
when I add  a pot direction where store my plot files, the dashboard reveal 'Failed to open (invalid plots)'
the situation can't be solved when I remove and add pot direction many times. I didnt open any others pool's app
I checked debug log "plot has a farmer public key that is not in the farmer's pk list."

**Comment by mrph28** [comment]
Bluepool doesn't support old plots yet. You have to replot using the bluepool GUI.

**Comment by kihcl** [comment]
got it!  thanks!
Can new pots which replot using bluepool GUI support officeial chia farming GUI?

**Comment by mrph28** [comment]
no, because the new plots are linked to a different key thats not the one on oficial chia GUI.

**Comment by chiabluepool** [comment]
Old plots are supported now, but only through a win/mac installer. This version is closed source because using your own keys makes it possible to cheat the pool by submitting partials that are not winning and keeping the ones that are winning. (This is the issue HPool had before it went on a banning spree)

**Comment by PuppyLover101** [comment]
Linux / Ubuntu when?
